lib/v2/tfbd/index.js

```diff
@@ -0,0 +1,30 @@
+const utils = require('./utils');
+const got = require('@/utils/got');
+const dateParser = require('@/utils/dateParser');
+
+module.exports = async (ctx) => {
+    const base = utils.urlBase(ctx.params.caty, ctx.params.id);
+    const res = await got.get(base, {
+        https: {
+            rejectUnauthorized: false,
```

Comment:
This option is only valid when you perform HTTPS requests. However, according to

https://github.com/DIYgod/RSSHub/blob/c0df7898c1b448bb00ce31f2d447476b8264dc70/lib/v2/tfbd/utils.js#L9

you're performing HTTP requests